README.md

```diff
@@ -27,35 +27,9 @@ Get the package:
 $ go get github.com/milosgajdos83/servpeek
 ```
 
-Example test file could look like this:
+You can find an example program in the [examples](https://github.com/milosgajdos83/servpeek/tree/master/examples) directory.
 
-```go
-package example
+## TODO
```

Comment:
I wouldn't remove all the examples. I would show one or two here and then point the people to the `examples/` folder. BTW, I think that just pointing like: `[examples](examples/)` will work, but... don't really trust me on this one.